Traceback (most recent call last):
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/agco-kun/.pyenv/versions/3.9.13/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/agco-kun/.local/share/virtualenvs/kzspace-ePujzJDD/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import scipy.fftpack

xf = np.linspace(0.0, 1.0/(2.0*delta), (N//2))
yf = scipy.fftpack.fft(y)

plt.plot(xf, 2.0/N * np.abs(yf[:N//2]))
plt.xlabel("Frequency")
plt.show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [3][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01mscipy[39;00m[38;5;21;01m.[39;00m[38;5;21;01mfftpack[39;00m
[1;32m      3[0m xf [38;5;241m=[39m np[38;5;241m.[39mlinspace([38;5;241m0.0[39m, [38;5;241m1.0[39m[38;5;241m/[39m([38;5;241m2.0[39m[38;5;241m*[39mdelta), (N[38;5;241m/[39m[38;5;241m/[39m[38;5;241m2[39m))
[1;32m      4[0m yf [38;5;241m=[39m scipy[38;5;241m.[39mfftpack[38;5;241m.[39mfft(y)

[0;31mModuleNotFoundError[0m: No module named 'scipy'
ModuleNotFoundError: No module named 'scipy'

